src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <ul class="nav__list">
+        <li class="nav__item nav__logo">
+          <a href="#home" class="nav__link">
+            <img class="headerlogo"
+            src="images/logo.png"
+            alt="header logo">
+          </a>
+        </li>
+        <nav class="nav__group">
+          <li class="nav__item apple">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item" data-qa="hover" >
```

Comment:
`add data-qa="hover" attribute to the 4th link for testing (Laptops & computers)` - add it to link, not to list item